librad/src/git/include.rs

```diff
@@ -30,6 +30,9 @@ use crate::{
     peer::PeerId,
 };
 
+/// Config key to reference generated include files in working copies.
+const GIT_CONFIG_PATH_KEY: &str = "include.path";
```

Comment:
```suggestion
pub const GIT_CONFIG_PATH_KEY: &str = "include.path";
```

Reply:
I think it would still make sense to have this public even though you're exposing the function publically. Say for example, someone wanted to test for this being set.